paasta_itests/environment.py

```diff
@@ -64,6 +64,8 @@ def _clean_up_chronos_jobs(context):
                 print "after_scenario: Job %s is present in chronos. Deleting." % job['name']
                 context.chronos_client.delete(job['name'])
             time.sleep(1)
+    if hasattr(context, 'jobs'):
```

Comment:
hm, this was to make sure that the jobs were sanitised between each scenario.

the 'setup' part of this is done in [here](https://github.com/Yelp/paasta/blob/88defcdedbbd82d4be659d6047cf07fbb5dd88e1/paasta_itests/steps/setup_steps.py#L116). Given it's in the 'given a working paasta cluster', which is done for each scenario (though that's not a formal requirement), it renders the code you've referenced pretty useless. Maybe I should leave this here just to be defensive (in which case, I should move the 'setup' part of it somewhere else.